cmd/provisioner-localpv/app/helper_blockdevice.go

```diff
@@ -218,3 +227,25 @@ func (p *Provisioner) deleteBlockDeviceClaim(blkDevOpts *HelperBlockDeviceOption
 	}
 	return nil
 }
+
+//
+func (p *Provisioner) removeFinalizer(blkDevOpts *HelperBlockDeviceOptions) error {
+	glog.Info("removing local-pv finalizer on the BDC")
+
+	bdc, err := blockdeviceclaim.NewKubeClient().
+		WithNamespace(p.namespace).
+		Get(blkDevOpts.bdcName, metav1.GetOptions{})
+	if err != nil {
+		return errors.Errorf("unable to get BDC %s for removing finalizer", blkDevOpts.bdcName)
+	}
+
+	// edit the finalizer in the copy of the BDC
+	bdc.Finalizers = util.RemoveString(bdc.Finalizers, LocalPVFinalizer)
+
+	// udpate the BDC with the new finalizer array
+	_, err = blockdeviceclaim.NewKubeClient().
```

Comment:
If there are no changes to be updated, does this function return error or success?

Reply:
The k8s update call, will succeed even if both objects are exactly same right? The `Update` in bdc is a wrapper so , it will succeed.